lib/config.js

```diff
@@ -307,7 +307,16 @@ config.start = function start() {
     // Wait until metrics has started before starting cluster control, because
     // the workers need to know what ephemeral port the statsd server is
     // listening on in order to send it metrics.
-    return control.start(config);
+    debug('cluster-control size: %j', config.size);
+    var size = config.size < 0 ? {} : { size: config.size };
+    var c = control.start(size);
```

Comment:
`c` is unused. Was it meant to be used a few lines down for `.fork()`?

If not, is there a chance we could change this to avoid using `size` as both an object and an int? Something like:
```js
if (config.size >= 0) {
  control.start({ size: config.size });
} else {
  // Fork -config.size workers, but don't restart them when they exit.
  // Mostly used as `-1` by strong-arc.
  control.start({});
  for (var size = 0; size > config.size; size--) {
    cluster.fork();
  }
}
```